python/doc/examples/data_analysis/graphics/plot_visualize_input_output_sample.py

```diff
@@ -0,0 +1,39 @@
+"""
+Visualize pairs between two samples
+===================================
```

Comment:
there is another example  "Visualize pairs":

http://openturns.github.io/openturns/latest/auto_data_analysis/graphics/plot_visualize_pairs.html?highlight=visualize+pairs

which is quite poor, maybe the 2 can be merged into a single one ?

Reply:
These two examples do not share the same material. In the first one we check the correlations inside a same sample and in the second one, we check the correlations between the margins of two input / output samples. Maybe it could be nice to let appear these two test cases in the example page to more efficiently guide the user?

Reply:
There are more appropriate examples to factor, sometimes associated with the `plotXvsY` function:
- [plot_sensitivity_sobol.html](https://openturns.github.io/openturns/latest/auto_reliability_sensitivity/sensitivity_analysis/plot_sensitivity_sobol.html)
- [plot_chaos_ishigami.html](https://openturns.github.io/openturns/latest/auto_meta_modeling/polynomial_chaos_metamodel/plot_chaos_ishigami.html)

Reply:
> There are more appropriate examples to factor, sometimes associated with the `plotXvsY` function:
> 
>     * [plot_sensitivity_sobol.html](https://openturns.github.io/openturns/latest/auto_reliability_sensitivity/sensitivity_analysis/plot_sensitivity_sobol.html)
> 
>     * [plot_chaos_ishigami.html](https://openturns.github.io/openturns/latest/auto_meta_modeling/polynomial_chaos_metamodel/plot_chaos_ishigami.html)

I think we need a multi output function to illustrate the behavior of the method. And ishigami is a scalar output function.

Reply:
okay, it is now done